docs/en/samples/dataflow_affinity.md

```diff
@@ -97,15 +98,15 @@ When DataLoad B is running,  you will find the the affinity of its pod including
 
 The DataFlow consists of DataLoad A and DataLoad B. DataLoad A requires running on GPU nodes through customized label `node.kubernetes.io/instance-type`, while DataLoad B requires running on nodes with the same label value as DataLoad A (GPU nodes).
 
+- If DataLoad C (runAfter B) requires the affinity of this label, DataLoad B also needs to set the annotation `fluid.io/dataflow.affinity.labels: node.kubernetes.io/instance-type`
+
```

Comment:
What about using the name `fluid.io/affinity-labels`? I think no need to add dataflow.

Reply:
ok, will fix.